internal/development.md

```
@@ -462,7 +462,42 @@ DELETE METADATA OF DOCUMENT 'bbe55942535e11f1bc5184ba59049aa3' KEYS '["key1", "k
 RAGFlow(user)> DROP METADATA STORE
 ```
 
-- List metadata
+- Get metadata
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)